assets/js/modules/analytics/common/account-create.js

```diff
@@ -0,0 +1,127 @@
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback, useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+import ProgressBar from '../../../components/progress-bar';
+import { trackEvent } from '../../../util';
+import TimezoneSelect from './account-create/timezone-select';
+import AccountField from './account-create/account-field';
+import PropertyField from './account-create/property-field';
+import ProfileField from './account-create/profile-field';
+import CountrySelect from './account-create/country-select';
+import { STORE_NAME, FORM_ACCOUNT_CREATE } from '../datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import { countryCodesByTimezone } from '../util/countries-timezones';
+import Data from 'googlesitekit-data';
+
+const { useDispatch, useSelect } = Data;
+
+export default function AccountCreate() {
+	const accountTicketTermsOfServiceURL = useSelect( ( select ) => select( STORE_NAME ).getAccountTicketTermsOfServiceURL() );
+	const canSubmitAccountCreate = useSelect( ( select ) => select( STORE_NAME ).canSubmitAccountCreate() );
+	const isDoingCreateAccount = useSelect( ( select ) => select( STORE_NAME ).isDoingCreateAccount() );
+	const siteURL = useSelect( ( select ) => select( CORE_SITE ).getReferenceSiteURL() );
+	const siteName = useSelect( ( select ) => select( CORE_SITE ).getSiteName() );
+	let timezone = useSelect( ( select ) => select( CORE_SITE ).getTimezone() );
+
+	const [ isNavigating, setIsNavigating ] = useState( false );
+
+	// Redirect if the accountTicketTermsOfServiceURL is set.
+	if ( accountTicketTermsOfServiceURL ) {
+		global.location.assign( accountTicketTermsOfServiceURL );
+	}
+
+	// Set form defaults on initial render.
+	const { setForm } = useDispatch( STORE_NAME );
+	useEffect( () => {
+		const { hostname } = new URL( siteURL );
+		timezone = countryCodesByTimezone[ timezone ] ? timezone : Intl.DateTimeFormat().resolvedOptions().timeZone;
+		setForm( FORM_ACCOUNT_CREATE, {
+			accountName: siteName,
+			propertyName: hostname,
+			profileName: __( 'All website traffic', 'google-site-kit' ),
+			countryCode: countryCodesByTimezone[ timezone ],
+			timezone,
+		} );
+	}, [ siteName, siteURL, timezone ] );
+
+	const { createAccount } = useDispatch( STORE_NAME );
+	const handleSubmit = useCallback(
+		async () => {
+			trackEvent( 'analytics_setup', 'new_account_setup_clicked' );
+			const { error } = await createAccount();
+
+			if ( ! error ) {
+				setIsNavigating( true );
+			}
+		},
+		[ createAccount, setIsNavigating ]
+	);
+
+	if ( isDoingCreateAccount || isNavigating ) {
+		return <ProgressBar />;
+	}
+
+	return (
+		<div>
+			<h3 className="googlesitekit-heading-4">
+				{ __( 'Create new Analytics account', 'google-site-kit' ) }
```

Comment:
@felixarntz shouldn't this be "Create _a_ new Analytics Account" ?

```suggestion
				{ __( 'Create a New Analytics Account', 'google-site-kit' ) }
```

Reply:
Not sure about capitalizing these words since they're not part of a name or anything. @adamsilverstein This copy was reviewed by @marrrmarrr already right?

Reply:
@felixarntz The case change here would only be if it's the title of the view. If it's not really a title, and just a heading, then the original casing is fine. I'm actually more concerned about the missing "a" there 😄

Reply:
Per follow-up convo with @marrrmarrr changing this to "Create your Analytics account".